requirements/prod.in

```diff
@@ -37,6 +37,7 @@ pycryptodomex
 pyparsing >= 2.1.5
 pyramid
 pyramid-exclog
+jinja2 < 3
```

Comment:
We use jinaj2 directly, it should be here in the first place.

Pinning to the version we support. 

Will update this in a separate PR.

Reply:
Might want to be careful about pinning things in the `*.in` files as I think it may prevent Dependabot PRs

Reply:
Yeah, the actual upgrade for it is here so we won't forget about this: https://github.com/hypothesis/h/pull/8434